accounts-management/keystore/geth/adapter.go

```diff
@@ -46,82 +48,134 @@ func NewGethKeystoreAdapter(keystoreDir string, scryptN int, scryptP int) (*Adap
 	}, nil
 }
 
+func porcessError(err error) error {
```

Comment:
it's not really processing it
```suggestion
func convertError(err error) error {
```

Reply:
True, update it to `mapToKeystoreError`.